### PR TITLE
Key maps and keys moved to their own module

### PR DESCRIPTION
**Pull request by KenjiBrown**

### CODE REVIEW

**Review by imcsk8 — APPROVED** [review]
LGTM

**Comment by KenjiBrown** [comment]
Aguanta. Todavía le falta.

File: src/functions.rs
Comment (zodman) [inline src/functions.rs:1]: `utils.rs` its better

Comment (KenjiBrown) [reply src/functions.rs:1]: Renamed fucntions to utils in 5793c2f859a62f114ee94433b29cbc4ec20da7c1

**Pull request merged**